pkg/beholder/global.go

```diff
@@ -12,7 +12,9 @@ import (
 )
 
 // Pointer to the global Beholder Client
-var globalClient = defaultClient()
+var globalClient atomic.Pointer[Client]
```

Comment:
This is to sync access within the `beholder` pkg? What consumers interact with `globalClient`?

Reply:
There is an exported `SetClient` function.